quantlib/termstructures/yields/rate_helpers.pyx

```diff
@@ -69,25 +69,25 @@ cdef class RelativeDateRateHelper:
 
 
 cdef class DepositRateHelper(RateHelper):
-    """Rate helper for bootstrapping over deposit rates. """
+    """Rate helper for bootstrapping over deposit rates. [uses SimpleQuotes] update 05/14/2015"""
```

Comment:
I would suggest moving the comment about SimpleQuotes to the changelog (https://github.com/enthought/pyql/blob/master/CHANGES.txt)